publish/rsync.sh

```diff
@@ -24,12 +29,12 @@ usage ()
 
   echo "To push JBT build + update site folders:"
   echo "   $0 -s \${WORKSPACE}/sources/aggregate/site/target/fullSite          -t mars/snapshots/builds/\${JOB_NAME}/\${BUILD_ID}-B\${BUILD_NUMBER}"
-  echo "   $0 -s \${WORKSPACE}/sources/aggregate/site/target/fullSite/all/repo -t mars/snapshots/updates/core/\${stream}"
+  echo "   $0 -s \${WORKSPACE}/sources/aggregate/site/target/fullSite/all/repo -t mars/snapshots/updates/core/\${stream} --del"
```

Comment:
"Some options require rsync to know the full file list, so these options
    disable the incremental recursion mode. These include: --delete-before, --delete-after, --prune-empty-dirs, and --delay-updates. Because of this, the default delete mode when you specify --delete is now --delete-during when both ends of the connection are at least 3.0.0 (use --del or --delete-during to request this improved deletion mode explicitly). See also the --delete-delay option that is a better choice than using --delete-after. "

-- http://linux.die.net/man/1/rsync

I picked the most efficient because it doesn't matter when the files are deleted. As soon as the new content.jar/artifacts.jar are pushed, all the old IUs become orphans, no longer accessible by p2.